original_codebase/data.py

```diff
@@ -341,35 +342,9 @@ def process(dataset: Iterable) -> list:
     return train_ds, test_ds
 
 
-def load_filipino(data_directory):
```

Comment:
this is the newer and correct version of loading Filipino dataset, we should keep that.

Reply:
I don't think this should be considered resolved yet, because the old parameterless version of `load_filipino`, which appears above this one, is still present, while this new version with a `data_directory` parameter is deleted. The old version is the one that retrieves from a source with 100% train-test leakage (see #13), while, as [noted](https://github.com/bazingagin/npc_gzip/pull/30/files#r1278224985), this is the fixed version.

Probably the opposite change should be made--the one above this one that is present should be deleted, while this should be restored. However, if the old broken `load_filipino` is to be kept, then I'd suggest renaming it (maybe to `load_filipino_hf`, I'm not sure) and also having it issue a warning when called. In either case, this currently deleted newer version should be restored.

Reply:
I think the older one could be deleted, just keep this one?

Reply:
Yes, I agree with deleting the old one and bringing back (i.e. undoing the deletion of) this one.

Reply:
Just to be clear, we're saying keep the version `def load_filipino(data_directory):`?

Reply:
Yes.